Apply filter to solar ability colours. (#8631)
DestinyItemManager/DIM: docs/CHANGELOG.md

  * Perk and mod tooltips for enhanced weapon traits and Exotic catalysts have unique styles to help them stand out.
  * The energy cost of armor mods is displayed within tooltips.
* Fixed an issue where the energy meter on Ghosts was not displaying the amount of energy that had been used by inserted mods.

> * Solar class ability and jump icons have been filtered to match other solar abilities (we couldn't handle it anymore).

Reviewer: ```suggestion
* Solar class ability and jump icons have had their colors adjusted to match other solar abilities (we couldn't handle it anymore).
```
